Convert applicable INFO logs to WARNING or SEVERE (#1950)

ssoloff opened the pull request:
As mentioned in [#1950](https://github.com/triplea-game/triplea/issues/1950#issuecomment-309639988), there are some `INFO` logs that are actually errors and should have their level increased to either `WARNING` or `SEVERE`.  The purpose of this PR is to identify those instances and increase their level accordingly before proceeding further with fixing #1950.

There are 50 instances of `INFO` logs in the codebase.  These were identified by searching for references to `Logger#info()` and `Logger#log(Level.INFO, ...)`.  The initial commit in this PR simply identifies those instances, and adds a comment immediately before it containing the proposed log level.  I believe most of the logs will remain at the `INFO` level.

Please review and comment on any proposed log levels you do not agree with.  Once an agreement has been reached, I'll update the PR accordingly.

**UPDATE:** All applicable `INFO` logs have been upgraded to either `WARNING` or `SEVERE`.  See [this comment](https://github.com/triplea-game/triplea/pull/2263#issuecomment-324822288) for additional changes that were made.

prastle commented:
This is a great idea! @ssoloff

ssoloff commented:
@prastle Well, it's really @RoiEXLab's and @ron-murhammer's idea.  I'm just executing it. :smile:

prastle commented:
Sigh i need to try and read more of this here ... :)

ron-murhammer commented on src/main/java/games/strategy/engine/lobby/server/db/Database.java:
```diff
@@ -260,6 +262,7 @@ private static void shutDownDB() {
       }
     }
+    // INFO
     logger.info("Databse shut down");
```
Typo here

ron-murhammer commented on src/main/java/games/strategy/engine/lobby/server/db/UserController.java:
```diff
@@ -45,6 +45,7 @@ public HashedPassword getPassword(final String userName) {
       return returnValue == null ? null : new HashedPassword(returnValue);
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
```
Probably should change this log message so it isn't a duplicate of the one below.

ron-murhammer commented on src/main/java/games/strategy/engine/lobby/server/db/UserController.java:
```diff
@@ -176,6 +178,7 @@ public DBUser getUserByName(final String userName) {
       return user;
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
```
Probably should change this error message as well.

ron-murhammer commented:
@ssoloff Added a few comments but pretty much agree with all your choices. Added comments where we should consider improving/fixing the log messages since we are looking at them.

DanVanAtta commented on src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java:
```diff
@@ -69,6 +70,7 @@ void removeBannedWord(final String word) {
       return rVal;
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error reading bad words error:" + sqle.getMessage());
```
Maybe remove instead? 
`IllegalStateException(sqle.getMessage());` should probably be: `IllegalStateException(sqle)`;

Logging and throwing indicates to me we are going into an unhandled crash. In that case, the stack trace should be logged. In that case the logging we are doing is redundant, and instead we could attach the extra message and propagate the exception eg:
``` throw new IllegalStateException("Error reading bad words error", sqle)```

ssoloff replied:
:+1: I was already in the process of implementing your first suggestion, but I think your second suggestion is the right thing to do and will update accordingly.

ssoloff commented:
I think this PR is now ready for review.  Both @ron-murhammer's and @DanVanAtta's suggestions have been implemented.

I also made the following changes:

* Converted calls to `Logger#log(Level.INFO, ...)` to `Logger#info()`.
* Renamed a few `Logger` fields that were not named `logger` simply to ensure consistency across the codebase.

I will update the PR title and description accordingly.
